ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py

```diff
@@ -2437,3 +2437,36 @@ def test_numpy_instance_abs__(
         frontend=frontend,
         frontend_method_data=frontend_method_data,
     )
+
+ 
+# __len__
+@given(
```

Comment:
Use @handle_frontend_method and include the additional parameters like class_tree, init_tree, method_name